wiki/heath_potion.md

```diff
@@ -0,0 +1,26 @@
+Turrets 
+=======
+
+* Icon: `♥`
+* Buy with: `l, 1, *`
+* Use with: `u, 1, *`
+* Cost: `50 Ore`
+
+#### Services Offered
+
+Health Potions give the user an instant boost of 15 health directly to their health bar.
+
+#### Owner Information
+
+Health Potions are acquired from the Pharmacy,
+a corporation owned building. They can be bought by anyone, with varying margins of profit. Members of a corporation (M),(@) gain you no profit. Allies (A) net you 1 ore, Neutrals (N) 5 ore, and Enemies (E) get you 10 ore per potion.
```

Comment:
For new sentences, just use a new line as it'll render the same but look neater when editing. Example:

`Line 1 - The bobcat is very silly.`
`Line 2 - The children loved to play with the bobcat, until it got fed up with their unclean smiles and mauled a child.`

renders into

`The bobcat is very silly. The children loved to play with the bobcat, until it got fed up with their unclean smiles and mauled a child.`